app/viewmodels/changeContactDetails/OrganisationContactPhoneSummary.scala

```diff
@@ -17,24 +17,27 @@
 package viewmodels.changeContactDetails
 
 import models.{CheckMode, UserAnswers}
-import pages.changeContactDetails.OrganisationPhonePage
+import pages.changeContactDetails.OrganisationContactPhonePage
 import play.api.i18n.Messages
 import play.twirl.api.HtmlFormat
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.checkAnswers.Util.changeAction
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-object OrganisationPhoneSummary {
+object OrganisationContactPhoneSummary {
 
   def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
-    answers.get(OrganisationPhonePage).map {
+    answers.get(OrganisationContactPhonePage).orElse(Some("Not provided")).map {
```

Comment:
So my understanding of interpretting this code is we should always show a row? Then if there is an answer we should show it, and if there is not an answer then we should show the text "Not provided"?

If my understanding is correct, I would be inclined to make a few changes:

* Change the method to return a `SummaryListRow` rather than an `Option[SummaryListRow]`
* Get the "Not provided" text from the messages file, rather than hard code it
* Reformat the code to get rid of the `.get().orElse().map` in favour of pulling the answer into a variable then returning the `SummaryListRowViewModel` - making it clear that it will always be returned.

I'm thinking something like this:

```
  def row(answers: UserAnswers)(implicit messages: Messages): SummaryListRow = {
    val telephone = answers.get(OrganisationContactPhonePage).getOrElse(messages("checkYourAnswers.notProvided"))

    SummaryListRowViewModel(
      key = s"$OrganisationContactPhonePage.checkYourAnswersLabel",
      value = ValueViewModel(HtmlFormat.escape(telephone).toString),
      actions = Seq(
        changeAction(
          OrganisationContactPhonePage.toString,
          controllers.changeContactDetails.routes.OrganisationContactHavePhoneController.onPageLoad(CheckMode).url
        )
      )
    )
  }
```